openapi/openapi.yaml

```diff
@@ -934,6 +934,30 @@ paths:
             default: false
           description: >
             Do not use cached results, always run adviser.
+        - name: github_event_id
+          in: query
+          required: false
+          schema:
+            type: string
+          description: : GitHub's event id.
+        - name: github_check_run_id
+          in: query
+          required: false
+          schema:
+            type: string
+          description: : GitHub's event id.
+        - name: github_installation_id
+          in: query
+          required: false
+          schema:
+            type: integer
+          description: GitHub's installation id.
+        - name: github_revision
```

Comment:
@pacospace to be consistent revision or commit_sha?

Reply:
Revision seems to be more general. I found that `"revision" refers to the id you can use as a parameter to reference an object in git (usually a commit)`. I would go for simple revision without github and the same for origin if in thamos language can refer to other than a repo_url. WDYT? @fridex @goern

Reply:
> Revision seems to be more general. I found that `"revision" refers to the id you can use as a parameter to reference an object in git (usually a commit)`. I would go for simple revision without github and the same for origin if in thamos language can refer to other than a repo_url. WDYT? @fridex @goern

+1 on the origin (that's why it was named "origin")

also +1 on revision

Reply:
Great! Let's adjust also descriptions then :) Thanks :beers:

Reply:
OK, please adjust thamos workflow part to be consistent. :beers:

Reply:
the "check_run" API we use needs `commit_sha` not a revision.

Reply:
But isn't revision a superset? as it can represent any git object, it includes also a commit_sha? Also in Argo workflow there is revision term.